Tests/OCTVideoEngineTests.m

```diff
@@ -145,4 +151,81 @@ - (void)testReceiveVideoFrame
     });
 }
 
+- (void)_performYUVCheckWithPlanes:(uint8_t * [3])planes strides:(OCTToxAVStrideData[3])strides
```

Comment:
Can you remove `_` from method name? So `_performYUVCheckWithPlanes` -> `performYUVCheckWithPlanes` :-)